app/views/choice_page.scala.html

```diff
@@ -14,41 +14,78 @@
  * limitations under the License.
  *@
 
-@import controllers.routes
-@import forms.Choice
+@import uk.gov.hmrc.govukfrontend.views.html.components._
 @import forms.Choice._
+@import forms.Choice
+@import config.AppConfig
 @import views.Title
-@import views.components.fields.RadioOption
 
-@this(main_template: views.html.main_template)
+@this(  govukLayout: views.html.gds_main_template,
+govukButton: GovukButton,
+govukRadios: GovukRadios,
+errorSummary: components.errorSummary,
+formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF,
+appConfig: AppConfig)
 
 @(form: Form[Choice])(implicit request: Request[_], messages: Messages)
 
-@main_template(title = Title("movement.choice.title", None)) {
-
-    @components.back_link(routes.StartController.displayStartPage())
-
-    @helper.form(routes.ChoiceController.submitChoice(), 'autoComplete -> "off") {
-        @helper.CSRF.formField
-
-        @components.error_summary(form.errors)
-
-        @components.heading(title = messages("movement.choice.title"))
-
-        @components.input_radio(
-            field = form("choice"),
-            legend = "",
-            inputs = Seq(
-                RadioOption("arrival", Arrival.value, messages(s"movement.choice.arrival.label")),
-                RadioOption("associate", AssociateUCR.value, messages(s"movement.choice.associate.label")),
-                RadioOption("disassociate", DisassociateUCR.value, messages(s"movement.choice.disassociateDucr.label")),
-                RadioOption("shut_mucr", ShutMUCR.value, messages("movement.choice.shutMucr.label")),
-                RadioOption("departure", Departure.value, messages(s"movement.choice.departure.label")),
-                RadioOption("submissions", Submissions.value, messages("movement.choice.submissions.label"))
-            )
-        )
+@govukLayout(
+    title = messages("movement.choice.title"),
+    backButtonTitle = messages("site.back"),
+    backButtonUrl = controllers.routes.StartController.displayStartPage().url) {
 
-        @components.submit_button(messages("site.continue"))
+    @formHelper(action = routes.ChoiceController.submitChoice(), 'autoComplete -> "off") {
+        @errorSummary(form.errors)
 
+        @govukRadios(Radios(
+            name = "choice",
+            fieldset = Some(Fieldset(
+                legend = Some(Legend(
+                    content = Text(messages("movement.choice.title")),
+                    isPageHeading = true,
+                    classes = "govuk-fieldset__legend--xl"
+                ))
+            )),
+            items = Seq(
+                RadioItem(
+                    id = Some(Arrival.toString.toLowerCase),
+                    value = Some(Arrival.value.toString),
+                    content = Text(messages(s"movement.choice.${Arrival.toString.toLowerCase}.label")),
+                    checked = form("choice").value.contains(Arrival.toString.toLowerCase)
```

Comment:
why we match against field not form bind value itself

Reply:
Awesome! :) Thank you!